### PR TITLE
Adding release 1.3 schedule to features repo

### PR DESCRIPTION
**Pull request by goltermann**
Adding this for parallelism to 1.4.  Will update other wikis to point here authoritatively.

@idvoretskyi @erictune

### CODE REVIEW

File: release-1.3/release-1.3.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+_(approved; last update 5/19)_
```
Comment (philips) [inline release-1.3/release-1.3.md]: approved by who?

**Comment by philips** [comment]
@goltermann I am not really familiar with this document or where it comes from. Could you add some context to your commit message? Thanks!

**Comment by goltermann** [comment]
Sorry, yeah, more details.  This is just a cut and paste from elsewhere.

We used to have this 1.3 doc under kubernetes/kubernetes/wiki/release-1.3.  The similar 1.4 doc was created under kubernetes/community/wiki/release-1.4.  Now we're removing them from the kubernetes and community repos to re-home them in the features repo, and under source control rather than wiki format.  I've updated those previous locations to point here.

The "approval" was just me marking the day it was presented in the community hangout and agreed on as plan of record.

**Comment by idvoretskyi** [comment]
@philips we have approved, but had zero bandwidth to commit. Thank you, @goltermann!

**Pull request merged**